temp.js

```diff
@@ -15,32 +15,32 @@ async function test() {
 
   try {
     // incogniaAPI.updateAccessToken()
-    const data = await incogniaAPI.registerOnboardingAssessment({
-      installationId:
-        'cUnIDfRgPyBnEU/2rzL32NSUV78yVVRHjPBaknJBQp+ZHb0tqbWpehqDqdmklyg6iS3gXG7asFsCjoTNlCE6BJJ8Vm/aKH7CrQTCqTUJsZdgI8YUe4Lze+F2lKXp4MPQfzs6varnIWT8wQQeJ/qJWA',
-      addressLine: 'Rua da Estrela, 77, Parnamirim, Recife, PE, 52060-282'
-    })
+    // const data = await incogniaAPI.registerSignup({
```

Comment:
Qual a ideia desse temp.js?
Ter algo pronto para testar?

Reply:
Acho que Xile deixou pra isso mesmo, bater de verdade na API. Tipo um teste de sanidade mesmo.

Reply:
Se for isso, que tal renomear?
Pensando que é uma lib pública, ver isso como temp.js é meio estranho

Reply:
ei po, é o endereço da casa da minha mãe que tá aí. Aí é foda

Reply:
Ninguém sabia, até tu dizer.  HUAUHAUHAUHAUHAUHAUHAUAUHUHA

Reply:
:joy:

Reply:
Bom deletar o comentário? Public PR

Reply:
Este temp.js tinha sido removido (foi mergido sem querer).